### PR TITLE
Add acknowledgements to sotd section.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
@msporny - Do we need to add the DHS etc acknowledgements to this spec as well? (Asking because of yesterday's tutorial.)

File: index.html [hunk 1]
```diff
@@ -93,22 +93,40 @@
...
+    reflect the position or the policy of the U.S. Government and no official
+    endorsement should be inferred.
+    </p>
+
+    <p class="issue">
+    Names in this section need to be updated:
+
+    Work on this specification has also been supported by the Rebooting the
+    Web of Trust community facilitated by Christopher Allen, Shannon Appelcline,
+    Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Manu Sporny,
```
Comment (msporny) [inline index.html]: The most up-to-date list is here:

Work on this specification has also been supported by the Rebooting the Web of Trust community facilitated by Christopher Allen, Shannon Appelcline, Kiara Robles, Brian Weller, Betty Dhamers, Kaliya Young, Kim Hamilton Duffy, Manu Sporny, Drummond Reed, Joe Andrieu, and Heather Vescent.

File: index.html [hunk 2]
```diff
@@ -93,22 +93,40 @@
...
+    <a href="mailto:public-credentials@w3.org">public-credentials@w3.org</a>
+    (<a href="mailto:public-credentials-request@w3.org?subject=subscribe">subscribe</a>,
+    <a href="https://lists.w3.org/Archives/Public/public-credentials/">archives</a>).
+    </p>
+
+    <p>
+    Portions of the work on this specification have been funded by the
+    United States Department of Homeland Security's Science and Technology
+    Directorate under contracts HSHQDC-16-R00012-H-SB2016-1-002 and
+    HSHQDC-17-C-00019. The content of this specification does not necessarily
```
Comment (msporny) [inline index.html]: Technically, only HSHQDC-17-C-00019 is paying for your time, @dmitrizagidulin. Unless @peacekeeper's time has been paid for by HSHQDC-16-R00012-H-SB2016-1-002 (Evernym) to work on this spec, I'd strike it.

**Comment by dmitrizagidulin** [comment]
@msporny Updated to reflect your comments. Let me know if there's anything else.

**Pull request merged**